### DISCUSSION TITLE
SOAR Integration

### DISCUSSION THREAD

**Discussion by fatmeat** [Security-Onion-Solutions/securityonion#9077] [Unsupported Versions]
Hello guys, do anyone know what solution should I integrate with security onion to archive auto block IP feature? eg. when a bad IP reputation alert triggered in security onion like pic below:
![image](https://user-images.githubusercontent.com/58129877/200183679-c4800142-4e57-4ea9-a4c9-5c290a5cd51f.png)

**Comment by 5MI7th3MI6** [comment]
It seems like they had it before https://www.youtube.com/watch?v=slCvK14NGwQ

This might help https://github.com/weslambert/DinoSOARLab
